nix/modules/tezos-accuser.nix

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  tezos-accuser-pkgs = {
+    "007-PsDELPH1" =
+      "${pkgs.tezos.tezos-accuser-007-PsDELPH1}/bin/tezos-accuser-007-PsDELPH1";
+    "008-PtEdo2Zk" =
+      "${pkgs.tezos.tezos-accuser-008-PtEdo2Zk}/bin/tezos-accuser-008-PtEdo2Zk";
+  };
+  cfg = config.services.tezos-accuser;
+in {
+  options.services.tezos-accuser = rec {
+    enable = mkEnableOption "Tezos accuser service";
+
+    node-name = mkOption {
+      type = types.str;
+      default = "edonet";
+      description = ''
+        Local name of the node, to support running multiple local instances.
```

Comment:
Umm, how is this supposed to work when you only have one optionset? (same with other modules)

E.g. someone imports the module, sets
```
services.tezos-accuser.node-name = "foobar";
```
How would they then add a second instance called `goodaz`? They can't just do
```
services.tezos-accuser.node-name = "goodaz";
```
because that would override the option and not magically create a second accuser instance.

What you probably want to do is move this out into a submodule and then the make the interface something like `services.tezos-accuser.instances.foobar = { <...> }` and `services.tezos-accuser.instances.goodaz = { <...> }` or drop this altogether and let the user handle this as for many other NixOS modules (e.g. using nixos-containers)